packages/valory/skills/apy_estimation_abci/models.py

```diff
@@ -192,9 +194,21 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
 
         self.__validate_params()
 
+    @property
+    def start(self) -> Optional[int]:
```

Comment:
`start` is now given by a property that utilizes the `ts_length`. The formula is:
```
start = end - n_observations * interval
```
end and interval are given in UNIX, and therefore start is also in UNIX.